iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/helpers/IotHubServicesCommon.java

```diff
@@ -27,6 +28,18 @@
 
     //if error injection message has not taken effect after 1 minute, the test will timeout
     private final static long ERROR_INJECTION_MESSAGE_EFFECT_TIMEOUT = 1 * 60 * 1000;
+    private final static String TEST_ASC_SECURITY_MESSAGE = "{ \"AgentVersion\": \"0.0.1\", "
+            + "\"AgentId\" : \"{4C1B4747-E4C7-4681-B31D-4B39E390E7F8}\", "
```

Comment:
>4C1B4747-E4C7-4681-B31D-4B39E390E7F8 [](start = 32, length = 36)

Is this a secret? If so, it should be an environment variable

Reply:
No it's not a secret, just a random guild I copied"

Reply:
Just a random guid.  This message doesn't mean anything.  I will make it look like more of a random guid.

---
In reply to: [322455012](https://github.com/Azure/azure-iot-sdk-java/pull/576#discussion_r322455012) [](ancestors = 322455012)